specs/wp-signup-spec.js

```diff
@@ -93,7 +94,17 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function() {
 		step(
 			'Can then see the domains page, and Can search for a blog name, can see and select a free .wordpress address in the results',
 			async function() {
-				const findADomainComponent = await FindADomainComponent.Expect( driver );
+				let findADomainComponent;
```

Comment:
I don't think we need to check for unavailability as this uses a free wordpress.com address which doesn't depend on the domains service

Reply:
Agree. I'll add checking only for paid domains and remove these.